pkg/disttask/framework/testutil/table_util.go

```diff
@@ -175,8 +175,10 @@ func GetSubtaskEndTime(ctx context.Context, mgr *storage.TaskManager, subtaskID
 
 // GetSubtaskNodes gets subtasks running nodes for one task for test.
```

Comment:
the comment should also be updated

Reply:
seems also fit, what you mean?

Reply:
I see the data is also queried from `tidb_background_subtask_history` so maybe

```suggestion
// GetSubtaskNodes gets nodes that are running or have run the task for test.
```